test/general/sycl_iterator.pass.cpp

```diff
@@ -3477,8 +3476,11 @@ DEFINE_TEST(test_includes)
     void
     operator()(Policy&& exec, Iterator1 first1, Iterator1 last1, Iterator2 first2, Iterator2 last2, Size n)
     {
-        TestDataTransfer<UDTKind::eKeys, Size> host_keys(*this, get_size(n));
-        TestDataTransfer<UDTKind::eVals, Size> host_vals(*this, get_size(n));
+        if (n < get_size())
```

Comment:
What does that comparison mean?

Reply:
This comparison mean that for this test some minimal buffer size required.